spec/models/manageiq/providers/nuage/network_manager/vcr_specs/refresher_spec.rb

```diff
@@ -2,6 +2,19 @@
   ALL_REFRESH_SETTINGS = [
     {
       :inventory_object_refresh => false
+    },
+    {
+      :inventory_object_refresh => true
```

Comment:
I haven't seen an example of just using the `inventory_object_refresh => true` setting. Do you perhaps know to what it defaults (the next one?)